Automattic/wp-calypso
# Plans redesign: Add new toggle design

#### Changes proposed in this Pull Request

This PR implements the experiment that redesigns the signup flow plans step. Users in the treatment group will see the new toggle section for monthly/annual plans.

NOTE: 
1. This PR uses the changes in https://github.com/Automattic/wp-calypso/pull/50040 as the base branch and is built on top of it.

**Important links**
1. [Figma design](https://www.figma.com/file/NhFdwnznk6LEuVtvc18qRm/LOHP-pricing-test?node-id=5%3A44)
2. pau2Xa-2yE-p2 for the project thread.


![Kapture 2021-02-18 at 18 17 31](https://user-images.githubusercontent.com/1269602/108359453-072a6d00-7216-11eb-94e9-ef8f7b6b3b06.gif)


#### Testing instructions



* Open http://calypso.localhost:3000/ and assign yourself to treatment of the `signup_plans_step_redesign_v1` experiment in the Abacus tool.
* Begin signup at `/start` and verify that the plans step UI matches the [Figma design](https://www.figma.com/file/NhFdwnznk6LEuVtvc18qRm/LOHP-pricing-test?node-id=5%3A44)
* Confirm that in Calypso's /plans view, the UI is unchanged.
* Also confirm that the launch flow UI is unchanged.
* Verify that control experience is unchanged.
 



Related to #
